### ISSUE TITLE
Possibility to make it work with .HAR files

### ISSUE THREAD

**Issue by m0t0r** [deshiknaves/cypress-msw-interceptor#2]
Hi, 

Thanks for this project. I am exploring and trying to make Cypress work together with this plugin for MSW: https://github.com/Tapico/tapico-msw-webarchive It works when MSW starts when the app is starting but doesn't work when I try to start MSW inside my Cypress tests, it registers all routes from .HAR file but does not actually return mocked responses. I was wondering if you are aware of that plugin and what's your opinion on that? Is it something we could try to make it work as well ?

Thanks

**Comment by deshiknaves** [comment]
Yep, I know of the plugin. This project is still in alpha and still a few things left on its core release functionality list. I'm currently waiting on a couple of PRs in MSW. Once the core features are stable, I can look into some of the extended features.
